### PR TITLE
chore: consolidate SortedSetPutElement to SortedSetElement, add util

### PR DESCRIPTION
**Pull request by cprice404**
This commit renames SortedSetPutElement to SortedSetElement, for
consistency with the Dictionary API.  Also introduces a helper function
for creating an array of SortedSetPutElements from a map, like we
support for Dictionaries.

### CODE REVIEW

File: responses/sorted_set_fetch.go [hunk 1]
```diff
@@ -48,6 +48,6 @@ func (r SortedSetFetchHit) ValueStringElements() []SortedSetStringElement {
 
 // ValueByteElements returns the elements as an array of objects, each containing a `value` and `score` field.
 // The value is a byte array, and the score is a number.
-func (r SortedSetFetchHit) ValueByteElements() []SortedSetElement {
+func (r SortedSetFetchHit) ValueByteElements() []SortedSetBytesElement {
```
Comment (poppoerika) [inline responses/sorted_set_fetch.go:51]: nit: We use `Byte` in `ValueByteElements` but we use `Bytes` in `SortedSetBytesElement`?

**Review by pgautier404 — APPROVED** [review]

File: responses/sorted_set_fetch.go (hunk 1 above)
Comment (poppoerika) [reply responses/sorted_set_fetch.go:51]: 🙇‍♀️

Comment (cprice404) [reply responses/sorted_set_fetch.go]: renamed in https://github.com/momentohq/client-sdk-go/pull/279/commits/4a635af8ddd1fa5886d16034c0583ed73819a1b7

**Review by pgautier404 — APPROVED** [review]

**Review by poppoerika — APPROVED** [review]
🚀

**Pull request merged**